spdx-utils/src/main/kotlin/SpdxLicenseAliasMapping.kt

```diff
@@ -133,7 +109,14 @@ object SpdxLicenseAliasMapping {
         "w3cl" to W3C,
         "wtf" to WTFPL,
         "zope" to ZPL_2_1
-    ).mapValues { (_, v) -> v.toExpression() }
+    ).mapValues { (_, v) -> v.toExpression() }.let { caseSensitiveMap ->
```

Comment:
Would you mind converting the remaining lowercase abbreviations to uppercase to make the file look more consistent?

Reply:
I'll probably simply convert all keys to upper or lower case in a follow-up commit.

Reply:
For those I would prefer to use normal capitalization, only changing e.g. "public domain..." to "Public Domain..." or "the Apache..." to "The Apache". But if we would convert them, I would prefer lowercase over uppercase.